domenetjenester/datavarehus/src/main/java/no/nav/foreldrepenger/datavarehus/v2/StønadsstatistikkUttakPeriode.java

```diff
@@ -44,7 +44,18 @@ enum Forklaring {
         MINSTERETT,
         FLERBARNSDAGER,
         //Samtidig
-        SAMTIDIG_MØDREKVOTE
+        SAMTIDIG_MØDREKVOTE,
+
+        AVSLAG_ANNET,
+        AVSLAG_AKTIVITETSKRAV,
+        AVSLAG_SØKNADSFRIST,
+        AVSLAG_IKKE_SØKT,
+        AVSLAG_UTSETTELSE,
+        AVSLAG_UTSETTELSE_TILBAKE_I_TID,
+        AVSLAG_PLEIEPENGER,
+        AVSLAG_STARTET_NY_STØNADSPERIODE,
+        AVSLAG_BARNETS_ALDER,
```

Comment:
Lurer på om vi kan slå sammen alder + stønadsperiode. Begge går på 14-10 tredje ledd.

Reply:
Kan være greit å kommentere med paragraf så det ser ut som vi har tenkt
AKTIVITETSKRAV er 14-13
Utsettelsene og IKKE_SØKT er 14-11 - kommentar på sammenhengende uttak for tilbake i tid
Stønadsperiode + Alder er 14-10 tredje ledd
PLEIEPENGER er 14-10 a prematurdager
Vilkår er 14-2 og 14-5
ANNET er diverse.

Vi kan ta med en egen AVSLAG for BFHR_IKKE_SØKT - 14-14 - for årsak 4102